extensions/Cake.DocCreator.yml

```diff
@@ -3,10 +3,11 @@ Name: Cake.DocCreator
 NuGet: Cake.DocCreator
 Assemblies:
 - "/**/Cake.DocCreator.dll"
-Repository: https://github.com/agc93/DocCreator/
+Repository: https://bitbucket.org/agc93/DocCreator/
 Author: agc93
 Description: A Cake addin to generate HTML documentation from Markdown source, using DocCreator
 Categories:
 - documentation
 - markdown
-TargetCakeVersion: 0.19.4
+TargetCakeVersion: 0.10.1
```

Comment:
Why the change here?

Reply:
Because we changed the logic to ensure we analyze the most recent "stable" release:

![image](https://user-images.githubusercontent.com/112710/101053121-681aa100-3555-11eb-9fb4-1e7be1b5d893.png)